base/install_motor.sh

```diff
@@ -34,3 +33,45 @@ cd ..
 
 make -j${JOBS}
 make clean
+
+#Add pmac module
+
+cd ..
+
+git clone --depth 1 --branch ${PMAC_VERSION} \
+    https://github.com/dls-controls/pmac
+
+cd pmac
+
+rm -rf .git/ .git*
+
+module_releases="PMAC=${EPICS_MODULES_PATH}/pmac
+"
+echo "$module_releases" >> ${EPICS_MODULES_PATH}/../RELEASE
+
+echo "
+" > configure/RELEASE.local.linux-x86_64
```

Comment:
I'd rather just `rm -f` it.

IMHO, we may remove any other `*linux*x86` file that influences the build. Special care with architecture-dependent configuration can be handled when we start testing `epics-in-docker` with each of them. For now, Intel x86-64 is our default.

Reply:
Eu tentei remover, mas foi gerado erro na compilação, vou procurar o arquivo que inclui o `RELEASE.local.linux-x86_64` e comentar esse trecho.